### PR TITLE
Remove Cori from documentation

### PR DESCRIPTION
**Pull request by RemiLehe**

### CODE REVIEW

File: Docs/source/install/hpc/perlmutter.rst [hunk 1]
```diff
@@ -285,9 +285,32 @@ Post-Processing
```
Comment (RemiLehe) [inline Docs/source/install/hpc/perlmutter.rst:273]: The changes below were moved over from the Cori documentation.

**Comment by ax3l** [comment]
Hehe, I eyed that as well :D

One wish: can you please copy over the documentation for Jupyter to Perlmutter? We currently only side- ref it and would loose it if we remove `cori.rst`

**Comment by ax3l** [comment]
Oh, you already did that :sparkling_heart:

File: Docs/source/install/hpc/perlmutter.rst [hunk 2]
```diff
@@ -285,9 +285,32 @@ Post-Processing
...
+When opening a Jupyter notebook, just select the name you picked for your custom kernel on the top right of the notebook.
+
+Additional software can be installed later on, e.g., in a Jupyter cell using ``!mamba install -c conda-forge ...``.
+Software that is not available via conda can be installed via ``!python -m pip install ...``.
+
+.. warning::
+
+   Jan 6th, 2022 (NERSC-INC0179165 and `ipympl #416 <https://github.com/matplotlib/ipympl/issues/416>`__):
+   Above, we fixated the ``ipympl`` version to *not* take the latest release of `Matplotlib Jupyter Widgets <https://github.com/matplotlib/ipympl>`__.
+   This is an intentional work-around; the ``ipympl`` version needs to exactly fit the version pre-installed on the Jupyter base system.
```
Comment (ax3l) [inline Docs/source/install/hpc/perlmutter.rst]: Let's verify the matching of `ipympl` versions is still needed?

We opened an upstream issue with the ipympl team the other year and they fixed that. If the base install is new enough on NERSC's Jupyter, than this should  be not a problem anymore :crossed_fingers: 
https://github.com/matplotlib/ipympl/issues/416

There is another issue https://github.com/matplotlib/ipympl/issues/458 that I do not fully understand right away, but NERSC says is needed, too.

File: Docs/source/install/hpc/perlmutter.rst [hunk 3]
```diff
@@ -273,9 +273,35 @@ Post-Processing
...
+As a one-time preparatory setup, `create your own Conda environment as described in NERSC docs <https://docs.nersc.gov/services/jupyter/#conda-environments-as-kernels>`__.
+In this manual, we often use this ``conda create`` line over the officially documented one:
 
-The Perlmutter ``$PSCRATCH`` filesystem is only available on *Perlmutter* Jupyter nodes.
-Likewise, Cori's ``$SCRATCH`` filesystem is only available on *Cori* Jupyter nodes.
-You can use the Community FileSystem (CFS) from everywhere.
+.. code-block:: bash
+
+   module load python
+   conda create -n myenv -c conda-forge
```
Comment (RemiLehe) [inline Docs/source/install/hpc/perlmutter.rst]:
```suggestion
   conda create -n myenv -c conda-forge mamba
```

File: Docs/source/install/hpc/perlmutter.rst [hunk 4]
```diff
@@ -273,9 +273,35 @@ Post-Processing
...
+   conda activate myenv
+   mamba install -c conda-forge python mamba ipykernel ipympl==0.8.6 matplotlib numpy pandas yt openpmd-viewer openpmd-api h5py fast-histogram dask dask-jobqueue pyarrow
```
Comment (RemiLehe) [inline Docs/source/install/hpc/perlmutter.rst]:
```suggestion
   mamba install -c conda-forge python ipykernel ipympl==0.8.6 matplotlib numpy pandas yt openpmd-viewer openpmd-api h5py fast-histogram dask dask-jobqueue pyarrow
```

Comment (ax3l) [reply Docs/source/install/hpc/perlmutter.rst]: Let' s keep mamba in here.

Also, we need to limit the upper version of matplotlib:
```suggestion
   mamba install -c conda-forge python==3.8 mamba ipykernel==6.13.0 ipympl==0.8.6 ipython==8.5.0 ipywidgets==7.7.0 jupyter-core==4.11.1 jupyter-server==1.18.1 jupyter_client==7.3.5 jupyterlab==3.4.2 jupyterlab-widgets==3.0.3 jupyterlab_server==2.15.1 "matplotlib<3.6" notebook==6.4.12 nbclient==0.6.4 numpy pandas "traitlets<6" yt openpmd-viewer openpmd-api h5py fast-histogram dask dask-jobqueue pyarrow
```
https://github.com/matplotlib/ipympl#install-an-old-jupyterlab-extension

- [ ] testing after PM is back online

File: Docs/source/install/hpc/perlmutter.rst [hunk 5]
```diff
@@ -273,9 +273,35 @@ Post-Processing
 
 For post-processing, most users use Python via NERSC's `Jupyter service <https://jupyter.nersc.gov>`__ (`Docs <https://docs.nersc.gov/services/jupyter/>`__).
 
-Please follow the same process as for :ref:`NERSC Cori post-processing <post-processing-cori>`.
-**Important:** The *environment + Jupyter kernel* must separate from the one you create for Cori.
+As a one-time preparatory setup, `create your own Conda environment as described in NERSC docs <https://docs.nersc.gov/services/jupyter/#conda-environments-as-kernels>`__.
+In this manual, we often use this ``conda create`` line over the officially documented one:
```
Comment (RemiLehe) [inline Docs/source/install/hpc/perlmutter.rst]:
```suggestion
As a one-time preparatory setup, create your own Conda environment:
```

File: Docs/source/install/hpc/perlmutter.rst [hunk 6]
```diff
@@ -273,9 +273,35 @@ Post-Processing
...
-Likewise, Cori's ``$SCRATCH`` filesystem is only available on *Cori* Jupyter nodes.
-You can use the Community FileSystem (CFS) from everywhere.
+.. code-block:: bash
+
+   module load python
+   conda create -n myenv -c conda-forge mamba
+   conda activate myenv
+   mamba install -c conda-forge python ipykernel ipympl==0.8.6 matplotlib numpy pandas yt openpmd-viewer openpmd-api h5py fast-histogram dask dask-jobqueue pyarrow
+   conda deactivate
+
```
Comment (RemiLehe) [inline Docs/source/install/hpc/perlmutter.rst:286]:
```suggestion
(For more info on `conda` environments at NERSC, see `this page <https://docs.nersc.gov/services/jupyter/#conda-environments-as-kernels>`__.)

```

File: Docs/source/install/hpc/perlmutter.rst [hunk 7]
```diff
@@ -273,9 +273,35 @@ Post-Processing
...
+In this manual, we often use this ``conda create`` line over the officially documented one:
 
-The Perlmutter ``$PSCRATCH`` filesystem is only available on *Perlmutter* Jupyter nodes.
-Likewise, Cori's ``$SCRATCH`` filesystem is only available on *Cori* Jupyter nodes.
-You can use the Community FileSystem (CFS) from everywhere.
+.. code-block:: bash
+
+   module load python
+   conda create -n myenv -c conda-forge mamba
+   conda activate myenv
```
Comment (RemiLehe) [inline Docs/source/install/hpc/perlmutter.rst]:
```suggestion
   conda create --yes -n warpx_postproc -c conda-forge mamba
   conda activate warpx_postproc
```

File: Docs/source/install/hpc/perlmutter.rst [hunk 8]
```diff
@@ -273,9 +273,35 @@ Post-Processing
...
+   mamba install -c conda-forge python ipykernel ipympl==0.8.6 matplotlib numpy pandas yt openpmd-viewer openpmd-api h5py fast-histogram dask dask-jobqueue pyarrow
```
Comment (RemiLehe) [inline Docs/source/install/hpc/perlmutter.rst]:
```suggestion
   mamba install --yes -c conda-forge python ipykernel ipympl==0.8.6 matplotlib numpy pandas yt openpmd-viewer openpmd-api h5py fast-histogram dask dask-jobqueue pyarrow
   python -m ipykernel install --user --name warpx_postproc --display-name WarpXPostProcessing
   echo '#!/bin/bash\nmodule load python\nsource activate warpx_postproc\nexec "$@"' > $HOME/.local/share/jupyter/kernels/warpx_postproc/kernel-helper.sh
```

File: Docs/source/install/hpc/perlmutter.rst [hunk 9]
```diff
@@ -273,9 +273,35 @@ Post-Processing
...
+
+.. code-block:: bash
+
+   #!/bin/bash
+   module load python
+   source activate myenv
+   exec "$@"
+
+When opening a Jupyter notebook, just select the name you picked for your custom kernel on the top right of the notebook.
+
```
Comment (RemiLehe) [inline Docs/source/install/hpc/perlmutter.rst:299]: (For more info on `conda` environments and JupyterHub at NERSC, see `this page <https://docs.nersc.gov/services/jupyter/how-to-guides/#how-to-use-a-conda-environment-as-a-python-kernel>`__.)

Comment (RemiLehe) [inline Docs/source/install/hpc/perlmutter.rst:302]:
```suggestion
(For more info on `conda` environments and JupyterHub at NERSC, see `this page <https://docs.nersc.gov/services/jupyter/how-to-guides/#how-to-use-a-conda-environment-as-a-python-kernel>`__.)

```

File: Docs/source/install/hpc/perlmutter.rst [hunk 10]
```diff
@@ -273,9 +273,35 @@ Post-Processing
...
+``kernel_helper.sh`` should read:
+
+.. code-block:: bash
+
+   #!/bin/bash
+   module load python
+   source activate myenv
+   exec "$@"
+
+When opening a Jupyter notebook, just select the name you picked for your custom kernel on the top right of the notebook.
```
Comment (RemiLehe) [inline Docs/source/install/hpc/perlmutter.rst]:
```suggestion
When opening a Jupyter notebook, just select `WarpXPostProcessing` from the list of available kernels on the top right of the notebook.
```

File: Docs/source/install/hpc/perlmutter.rst [hunk 11]
```diff
@@ -273,9 +273,35 @@ Post-Processing
...
+We then follow the `Customizing Kernels with a Helper Shell Script <https://docs.nersc.gov/services/jupyter/#customizing-kernels-with-a-helper-shell-script>`__ section to finalize the setup of using this conda-environment as a custom Jupyter kernel.
+
+``kernel_helper.sh`` should read:
+
+.. code-block:: bash
+
+   #!/bin/bash
+   module load python
+   source activate myenv
+   exec "$@"
```
Comment (RemiLehe) [inline Docs/source/install/hpc/perlmutter.rst]:
```suggestion
```

File: Docs/source/install/hpc/perlmutter.rst (hunk 2 above)
Comment (ax3l) [reply Docs/source/install/hpc/perlmutter.rst]: Since Oct 31st, NERSC rolled out the new ipympl base version :tada:

**Comment by ax3l** [comment]
Thank you!

I push a few updates in a minute and finalize testing :)

**Review by ax3l — APPROVED** [review]
Ready steady goooo :rocket: :sparkles:

**Pull request merged**